x-pack/docs/en/rest-api/security/app-privileges.asciidoc

```diff
@@ -0,0 +1,285 @@
+[role="xpack"]
+[[security-api-app-privileges]]
+=== Application Privileges APIs
+
+The Application Privileges API enables you to add, remove, and retrieve
+{xpack-ref}/defining-roles.html#roles-application-priv[Application Privileges].
+
+==== Request
+
+`GET /_xpack/security/privilege` +
+
+`GET /_xpack/security/privilege/<application>` +
+
+`GET /_xpack/security/privilege/<application>/<privilege>` +
+
+`DELETE /_xpack/security/privilege/<application>/<privilege>` +
+
+`POST /_xpack/security/privilege/` +
+
+`POST /_xpack/security/privilege/<application>/<privilege>` +
+
+`PUT /_xpack/security/privilege/<application>/<privilege>`
+
+
+==== Description
+
+Application privileges allow Elasticsearch to act as an _oracle_ for security
```

Comment:
Is this use of the term "oracle" well known? It didn't seem obvious when I search on that term.

Reply:
No, I don't think it's well enough known, but unfortunateIy couldn't find a better word. Any suggestions?
Probably I just need to restructure this sentence to not need a noun to describe ES's role here.